src/components/Card/_mixins.scss

```diff
@@ -17,7 +25,8 @@ $card__name: card;
     $count: 4,
   );
 
-  @include input__background-color;
+  // TODO: Modular styles - Re-enable once the mixin is fixed
+  // @include input__background-color;
```

Comment:
Should probably add a note to #44 about this in case we don't have one already. Or mention it in an issue that tackles global styles (rather than the component styles effort we're working on at the moment).